working/macros/api/introspection.dart

```diff
@@ -128,10 +139,7 @@ abstract class FieldDeclaration implements VariableDeclaration {
 }
 
 /// Parameter introspection information.
-abstract class ParameterDeclaration {
-  /// The name of the parameter.
-  String get name;
-
+abstract class ParameterDeclaration implements Declaration {
```

Comment:
No more name? Is it inherited from Declaration?

What about required-ness and default values?

Reply:
> No more name? Is it inherited from Declaration?

Yes I made it implement `Declaration` instead which has a name, so it gets it from that.

> What about required-ness and default values?

Ya it has those (you need to expand a section below).